.github/PR_TEMPLATE.md

```diff
@@ -0,0 +1,15 @@
+Thanks for your interest in plotly.js!
```

Comment:
Oh. I think this file needs to be named `PULL_REQUEST_TEMPLATE.md` according to https://github.com/blog/2111-issue-and-pull-request-templates